src/styles/blocks/card-variables.scss

```diff
@@ -0,0 +1,5 @@
+$main-gray: #616070;
+$color-batton: #00acdc;
```

Comment:
The variable name `$color-batton` seems to be a typo. It should likely be `$color-button` to accurately describe its purpose and maintain consistency in naming conventions.